test/pummel/test-hash-seed.js

```diff
@@ -1,20 +1,32 @@
 'use strict';
 
 // Check that spawn child doesn't create duplicated entries
-require('../common');
+const common = require('../common');
 const REPETITIONS = 2;
 const assert = require('assert');
 const fixtures = require('../common/fixtures');
-const { spawnSync } = require('child_process');
+const { spawn } = require('child_process');
 const targetScript = fixtures.path('guess-hash-seed.js');
 const seeds = [];
 
+const requiredCallback = common.mustCall(() => {
+  console.log(`Seeds: ${seeds}`);
+  assert.strictEqual(new Set(seeds).size, seeds.length);
+  assert.strictEqual(seeds.length, REPETITIONS);
+});
+
+let soFar = 0;
```

Comment:
Could this be replaced with `common.countdown`?

Reply:
¯\\_(ツ)\_/¯

To be honest I found this test hard to read regardless because you need to understand the fixture. With regards to `countdown` the changes seemed halfway there with the logic extracted to `requiredCallback`. I was mainly suggesting it for consideration, so having considered it feel free to leave it the way it is in this PR 😁.

Reply:
I went with countdown after all.